app/assets/javascripts/admin/helpers/smart-poller.js

```diff
@@ -0,0 +1,19 @@
+(function (GOVUK) {
+  'use strict'
+
+  GOVUK.smartPoller = function (wait, polledFunction) {
```

Comment:
Is this used anywhere else?

We've generally moved away from attaching things to the global GOVUK scope. So it might be worth considering moving this into the module itself

Reply:
Had a look and bootstrap only used it for link checking so have rolled it in 👍